modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -324,12 +250,88 @@ module.exports = {
       },
       // Add on the core default options to use, if needed.
       getBrowserData(_super, req) {
-        const result = _super(req);
+        const initialData = _super(req);
 
-        _.defaults(result, {
+        const rteData = {
+          components: {
+            widgetEditor: 'AposRichTextWidgetEditor',
+            widget: 'AposRichTextWidget'
+          },
+          tools: {
```

Comment:
Same comment for tools.

Reply:
As above, I can move this back into the options in its own `tools` option.